Sources/AppStoreConnectCLI/Arguments/Identifier.swift

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+enum Identifier {
```

Comment:
Identifier here is not very clear
```suggestion
enum App.Identifier {
```

Reply:
It can be an app id or a bundle id..so I am not sure whether calling it App.Identifier is justified or misleading.. should I rename it AppOrBundleIdentifier or something like that?

Reply:
I think `App.Identifier` is all right, either bundle Id or app Id is actually the unique identifier for an App, so you can either use bundle Id or app Id to identify an app resource. Can wait for @adamjcampbell for another look on this :)

Reply:
True you can use app id or bundle id to identify an app but App.Identifier may be also read as an expanded form of appID. ..just by looking at the name App.IDentifier it doesn't jump out that  it can also be a bundle id..

Reply:
🤔 I agree that `AppIdentifier` is more specific than just `Identifier`. We do run the risk of having more name clashes. The intent of this object is actually that we use it to "choose how to find/lookup an app"  

So maybe we could work along those lines, `AppLookupIdentifier` or something, still sounds odd. But it does convey intent would pair with `AppLookupOptions` which sounds okay.

Let's go `AppIdentifier` for now maybe and come back to it

Reply:
AppLookupIdentifier sounds nice

Reply:
I will change it to AppLookUpIdentifier

Reply:
Lookup is defined as a single word so we can use the casing `AppLookupIdentifier` as opposed to `AppLookUpIdentifier` separating the words conveys a slightly different meaning

Reply:
ok. Should I also change IdentifierArgument and IdentifierOptions to AppLookupArgument and AppLookupOptions?

Reply:
@adamjcampbell done..please review..thx